advanced/helm/ei-pattern-1/templates/analytics-worker-deployment-2.yaml

```diff
@@ -0,0 +1,210 @@
+# Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
@Aaquiff please update the license header year to the current year.

Applies to all occurrences in the PR.

Reply:
Updated